### PR TITLE
Functions to get dff data for a given cell_roi_id

### PR DESCRIPTION
**Pull request by dougollerenshaw**
Adds functions to get dff traces for a given cell_roi_id without having to open the OphysExperimentObject

Goes directly to LIMS to get well_known_files, then opens the `OphysDffTraceFile` for the associated experiment

Example use:

```python
from visual_behavior.data_access import loading

cell_roi_id = 1080867088
dff = loading.get_dff_traces_for_roi(cell_roi_id)
```

I also had some problems with old tests that were using the scipy `medfilt` function. It seems that newer versions of scipy use `median_filter` instead, and the syntax has changed. Updating syntax broke a couple of tests on the old-format foraging files, but since these are no longer relevant, I simply removed the tests.

### CODE REVIEW

File: visual_behavior/data_access/from_lims.py [hunk 1]
```diff
@@ -2214,6 +2214,53 @@ def load_objectlist(ophys_experiment_id):
     return objectlist_dataframe
 
 
+def get_well_known_file_path(id, attachable_type, asset_name):
```
Comment (DowntonCrabby) [inline visual_behavior/data_access/from_lims.py:2217]: Does this work for all types of well known files?

Comment (DowntonCrabby) [reply visual_behavior/data_access/from_lims.py:2217]: Second question, does the filepath it returns work for different operating systems?

Comment (dougollerenshaw) [reply visual_behavior/data_access/from_lims.py:2217]: Yes, it returns works for all well known file types. For example:
```
from_lims.get_well_known_file_path(951410079, 'OphysSession', 'OphysRigSync')
```
Returns:

> '/allen/programs/braintv/production/neuralcoding/prod0/specimen_840355567/ophys_session_951410079/951410079.h5'

and 
```
from_lims.get_well_known_file_path(964440177, 'OphysExperiment', 'OphysMotionXyOffsetData')
```
Returns:

> '/allen/programs/braintv/production/visualbehavior/prod2/specimen_887127198/ophys_session_963496285/ophys_experiment_964440177/processed/964440177_suite2p_rigid_motion_transform.csv'

Note that those are the paths exactly as they are returned by LIMS. They are linux compatible, but not Windows compatible. I'll go ahead and add some code to prepend a slash and make it work for Windows.

Comment (DowntonCrabby) [reply visual_behavior/data_access/from_lims.py:2217]: There is a function in utils that should do this. 

utils.correct_filepath(filepath)

Comment (dougollerenshaw) [reply visual_behavior/data_access/from_lims.py:2217]: See my latest commit. It was just a matter of prepending an extra slash.

**Review by DowntonCrabby — APPROVED** [review]
Looks good

**Pull request merged**